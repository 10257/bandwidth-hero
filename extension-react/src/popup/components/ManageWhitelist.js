import React from 'react'
import { Form, TextArea } from 'semantic-ui-react'

export default ({ whitelist = [] }: { whitelist?: string[] }) => {
  return (
    <Form>
      <Form.Field control={TextArea} rows={4} value={whitelist.join('\n')} />
    </Form>
  )
}
