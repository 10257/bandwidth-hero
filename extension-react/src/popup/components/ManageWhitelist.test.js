import React from 'react'
import ManageWhitelist from './ManageWhitelist'
import renderer from 'react-test-renderer'

it('renders whitelist management correctly', () => {
  const tree = renderer.create(<ManageWhitelist />).toJSON()
  expect(tree).toMatchSnapshot()
})
