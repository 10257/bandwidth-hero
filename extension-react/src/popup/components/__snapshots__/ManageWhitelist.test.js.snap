// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders whitelist management correctly 1`] = `
<form
  className="ui form"
>
  <div
    className="field"
  >
    <textarea
      onChange={[Function]}
      required={undefined}
      rows={4}
      type={undefined}
      value={undefined}
    />
  </div>
</form>
`;
